config/confighttp/confighttp.go

```diff
@@ -397,7 +397,12 @@ func responseHeadersHandler(handler http.Handler, headers map[string]configopaqu
 
 // CORSSettings configures a receiver for HTTP cross-origin resource sharing (CORS).
 // See the underlying https://github.com/rs/cors package for details.
-type CORSSettings struct {
+// Deprecated: [v0.94.0] Use CORSConfig instead
+type CORSSettings CORSConfig
```

Comment:
This should be an alias so both types are interoperable 

```suggestion
type CORSSettings = CORSConfig
```